### ISSUE TITLE
AVS

### ISSUE THREAD

**Issue by IlseFerris** [Hennepin-County/MAXIS-scripts#140]
Script support needed for users to document the AVS form details and submitting/processing the request in the AVS System. 

Process maps have been created for both processes. Additional support could include:

- Building a DAIL scrubber to support TIKL's
- Add help buttons re: who needs to sign the forms
- Add help buttons re: who needs to submit AVS for

**Comment by C-Love** [comment]
Possible add this functionality into Docs Recvd.

**Comment by C-Love** [comment]
After this script is completed, we won't add it to 'Complete List of Scripts', instead once the script is done, we will create a new issue to incorporate this into Docs Received.

**Comment by C-Love** [comment]
@IlseFerris - I updated this code to remove the part where it determines if HC is either active, pending, or REIN because Kerry uses this functionality on already denied HC cases to ensure the detail is in the notes. 

She wants to have more discussion about this functionality going forward but for testing and because today is busy - I took the liberty so this project can keep moving.  Specifically I commented out lines 110 - 116.

**Comment by IlseFerris** [comment]
Awesome, thank you @C-Love for supporting Kerry and to keep this moving along!

**Comment by IlseFerris** [comment]
Script has been released to all users today. #510 ADMIN - AVS SUBMITTED has retired as this is the more comprehensive script. Will stay in end user testing for at least another week or two. Hot Topic will be created for 07/06/2021.

**Comment by IlseFerris** [comment]
Communication coming out in Hot Topics on July 5th. Script has been in end user testing since May 12. Successfully closing issue.
